test/requests/meals/destroy.test.js

```diff
@@ -0,0 +1,82 @@
+var app = require('../../../app');
+var request = require("supertest");
+var expect = require('chai').expect;
+var Food = require('../../../models').Food;
+var Meal = require('../../../models').Meal;
+var MealFood = require('../../../models').MealFood;
+
+describe('api v1 meals meal_id foods DELETE', function () {
+  describe('user can remove a food from meal in database', function () {
+    it('returns 204 upon deletion', (done) => {
+      Food.bulkCreate([
+        {
+          id: 1,
+          calories: "10",
+          name: "peas"
+        },
+        {
+          id: 2,
+          calories: "300",
+          name: "candy"
+        }
+      ]).then(() => {
+        return Meal.bulkCreate([
+          {
+            id: 1,
+            name: "breakfast"
+          },
+          {
+            id: 2,
+            name: "lunch"
+          }
+        ])
+      }).then(() => {
+        return MealFood.bulkCreate([
+          {
+            FoodId: 1,
+            MealId: 1
+          },
+          {
+            FoodId: 2,
+            MealId: 1
+          },
+          {
+            FoodId: 1,
+            MealId: 2
+          }
+        ])
+      }).then(() => {
```

Comment:
What's the purpose of finding the MealFood before deleting it? Kinda weird too because you lookup by MealId but there are two MealFoods with that MealId.

Reply:
Resolved on branch